transfer.go

```diff
@@ -20,6 +20,34 @@ type TransferSourceType string
 // "account_frozen", "could_not_process", "bank_account_restricted", "invalid_currency".
 type TransferFailCode string
 
+// TransferDestinationType consts represent valid transfer destinations.
+type TransferDestinationType string
+
+const (
+	// TransferDestinationBankAccount is a constant representing a transfer destination
+	// which is a bank account.
+	TransferDestinationBankAccount TransferDestinationType = "bank_account"
+
+	// TransferDestinationCard is a constant representing a transfer destination
+	// which is a card.
+	TransferDestinationCard TransferDestinationType = "card"
+
+	// TransferDestinationAccount is a constant representing a transfer destination
+	// which is a Stripe account.
+	TransferDestinationAccount TransferDestinationType = "account"
```

Comment:
Tiny nit, but mind just moving this up above bank account so we can have it alphabetized?